first_year_scraper/requirements_conda.txt

```diff
@@ -0,0 +1,117 @@
+# This file may be used to create an environment using:
```

Comment:
@iakshat Please help me understand how this file is used.

Reply:
@thealphadollar To be honest, I really don't know why the env export for conda is so messy. I am doing these env exports for the first time so I might have done something improper. This export was meant for people using conda to manage envs as the requirements.txt for pip and conda differ a bit in syntax, so I tried making it easy😋. 
Although we can move forward with the one for pip only as that is more commonly used by us.

Reply:
Yeah, as you have suggested, keeping one is better. It'll allow us to maintain only one. Since a lot of users use only pip, eventually in future we might stop, by accident, maintaining conda and it might cause problems. 

Thanks for the understanding.

Reply:
Ohh cool! I never thought of both getting out of sync. True that can cause us many problems. I'll change it accordingly!